packages/modeling/src/operations/extrusions/extrudeLinearPath2.js

```diff
@@ -0,0 +1,23 @@
+const { geom2, path2 } = require('../../geometries')
+
+const extrudeLinearGeom2 = require('./extrudeLinearGeom2')
+
+/*
+ * Extrude the given geometry using the given options.
+ *
+ * @param {Object} [options] - options for extrude
+ * @param {Array} [options.offset] - the direction of the extrusion as a 3D vector
+ * @param {Number} [options.twistAngle] - the final rotation (RADIANS) about the origin
+ * @param {Integer} [options.twistSteps] - the number of steps created to produce the twist (if any)
+ * @param {path2} geometry - the geometry to extrude
+ * @returns {geom3} the extruded 3D geometry
+*/
+const extrudePath2 = (options, geometry) => {
+  if (!geometry.isClosed) throw new Error('extruded path must be closed')
```

Comment:
I wonder which is better... throwing an error or closing the path if not already (clone)?

If throw then a comment about 'only closed paths are accepted' would be helpful in extrudeLinear() documentation.